### PR TITLE
Integration into Cyberbotics web site

### PR DESCRIPTION
**Pull request by omichel**
This PR fixes all the problems related with the integration of the doc on Cyberbotics web site.
Currently everything is rendered nicely on the web site. We can even view other branches very easily (see https://github.com/omichel/webots-doc/blob/gh-pages/README.md).
I merged the `style.css` into `main.css` which is the main css file of Cyberbotics' web site and committed that file.
However, I didn't yet tested if the local rendering was affected by these changes.

### CODE REVIEW

**Comment by omichel** [comment]
The local doc viewer works again and the online view is already online at: https://www.cyberbotics.com/doc/guide

:+1:

**Comment by fabienrohrer** [comment]
On Chrome, the rendering is not working as expected, while it's working on Firefox:

capture d ecran 2016-04-28 a 13 43 13

**Comment by fabienrohrer** [comment]
Could you explain the modifications on the CSS?

**Comment by fabienrohrer** [comment]
I'm not very confident with the merge of the 2 CSS, but I don't have good alternatives to propose.
Notably I fear that we will always tempt to drop some CSS rules vital for the Cyberbotics website.

**Comment by fabienrohrer** [comment]
.. and the result is very encouraging :-)

**Comment by omichel** [comment]
Well, it works on my Chrome (under Windows 10):
chrome
Can you try to empty your cache and try again?
Do you get any error message in the Javascript console?

**Comment by omichel** [comment]
About the CSS, I hesitated to split it into two files, namely main.css would remain the main css of Cyberbotics web site and doc.css would contain the CSS used by the webots-doc only. However, this is not a good idea as the two CSS may conflict with each other anyway (e.g., overriding rules, etc.). So having everything in the same file is better to avoid such conflicts. We could improve things a little bit by defining a class that would inherit to all the documentation elements, e.g., <div id="view" class="webots-doc"> and have the CSS use this class to restrict the styles and avoid conflicts with the rest of the CSS.

**Comment by fabienrohrer** [comment]
The cache was indeed responsible of my issues, it's now working as expected on Chrome too.

**Comment by fabienrohrer** [comment]
The dynamic load of the page seems broken on the Cyberbotics website.

**Comment by fabienrohrer** [comment]
Ok for the CSS.

File: js/viewer.js [hunk 1]
```diff
@@ -292,21 +333,32 @@ function receiveMenuContent(menuContent) {
...
-
     updateSelection();
 }
 
+function updateMenuScrollbar() {
+    var e = document.documentElement;
+    var t = document.documentElement.scrollTop || document.body.scrollTop;
+    var p = e.scrollHeight - t - e.clientHeight;
+    // console.log("height = " + e.scrollHeight + " scrollTop = " + t + " clientHeight = " + e.clientHeight);
+    if (p < 244)
```
Comment (fabienrohrer) [inline js/viewer.js]: There are several hard coded integers in this function, and it seems important to document their origin.

**Comment by fabienrohrer** [comment]
Minor note:
About coding style, and curly brackets on the one line conditions, I was following the convention we decided to adopt:
https://github.com/omichel/webots/wiki/Coding-style#javascript
I guess the reason of this choice is the tolerance to syntax error due to the fact that the language is interpreted.
However I admit I didn't follow the convention perfectly..

**Comment by fabienrohrer** [comment]
Ok, the online dynamic load is fixed thanks to a4f0f3c.
It's still broken in the offline version.

**Comment by omichel** [comment]
About coding styles, it is a pain to have different coding styles for C/C++ and for Javascript. For example, the use of curly brackets on one line conditions and the 4 spaces for tabulation which we don't use in C/C++... I would suggest to change this to have more consistent coding styles, e.g., to have a 2 spaces tabulation and no curly brackets for one line conditions in Javascript, like we do in C/C++.

**Comment by fabienrohrer** [comment]
I'm fixing the `setupUrlLocal()` function

**Comment by fabienrohrer** [comment]
I fixed `setupUrlLocal()` and finally rewrite `setupUrlOnline()` using RegExp too for homogeneity (and it offers a better expression power: the arguments may be inverted for example). The dynamic loading is working again locally and online.

Remains to fix the anchors links in the current page.

**Comment by fabienrohrer** [comment]
Ok, everything seems to be fixed now. dynamic load and anchors are working online and offline.

**Comment by fabienrohrer** [comment]
I updated the viewer.js on the FTP.

**Comment by fabienrohrer** [comment]
👍

**Comment by omichel** [comment]
:+1:

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**